### PR TITLE
Adding refactoring of packaging-utility

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE FHIR SERVICE

Slack / Refactoring
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [X] Required tests are included.
* [X] No extemporaneous files are included (i.e Complied files or testing results).
* [X] This PR is merging into the **correct branch**.
* [X] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleController.java [hunk 1]
```diff
@@ -106,7 +106,7 @@ public ResponseEntity<String> saveMeasure(
         MediaType.APPLICATION_XML_VALUE
       },
       consumes = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<StreamingResponseBody> generateMeasureExport(
+  public ResponseEntity<byte[]> generateMeasureExport(
```
Comment (nmorasb) [inline src/main/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleController.java:109]: not sure if I missed something like this on the other PR, but do we lose any performance here by switching from `StreamingResponseBody` to `byte[]`? Like something related to holding up the servlet container thread?

File: src/test/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleControllerMvcTest.java [hunk 2]
```diff
@@ -192,9 +192,9 @@ public void testExportMeasure() throws Exception {
     String madieMeasureJson =
         getStringFromTestResource("/measures/SimpleFhirMeasureLib/madie_measure.json");
 
-    doNothing()
-        .when(exportService)
-        .createExport(any(Measure.class), any(), any(Principal.class), anyString());
+    //    doNothing()
+    //        .when(exportService)
+    //        .createExport(any(Measure.class), any(Principal.class), anyString());
```
Comment (nmorasb) [inline src/test/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleControllerMvcTest.java]: probably remove this commented code

Comment (gregory-akins) [reply src/test/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleControllerMvcTest.java]: Done

File: src/main/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleController.java (hunk 1 above)
Comment (gregory-akins) [reply src/main/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleController.java:109]: Will research and resolve in an upcoming change.

Comment (gregory-akins) [reply src/main/java/gov/cms/madie/madiefhirservice/resources/MeasureBundleController.java:109]: Will research and resolve in an upcoming change.

**Review by nmorasb — APPROVED** [review]

**Pull request merged**